src/apify_client/clients/resource_clients/user.py

```diff
@@ -31,6 +33,55 @@ def get(self: UserClient) -> dict | None:
         """
         return self._get()
 
+    def monthly_usage(self: UserClient) -> dict | None:
+        """Return monthly usage of the user account.
+
+        This includes a complete usage summary for the current usage cycle, an overall sum, as well as a daily breakdown
+        of usage. It is the same information which is available on the account's Billing page. The information includes
+        use of storage, data transfer, and request queue usage.
+
+        https://docs.apify.com/api/v2/#/reference/users/monthly-usage
+
+        Returns:
+            dict, optional: The retrieved request, or None, if it did not exist.
```

Comment:
I'd just remove the `dict, optional:` from the docstrings, it's redundant since it's already contained in the type hint. For every endpoint.

Reply:
Good point. Removed in the whole file.